layers/+lang/latex/funcs.el

```diff
@@ -20,6 +20,14 @@
     ;; (when build-proc
     ;;   (set-process-sentinel build-proc 'latex//build-sentinel))))
 
+(defun latex/all ()
+  "Save, build and view in one command."
+  (interactive)
+  (progn
+    (let ((TeX-save-query nil))
+      (TeX-save-document (TeX-master-file)))
```

Comment:
Note that `(TeX-master-file)` may be a different file than the file the user is editing (which is presumably the one that is not saved).